platform/mds/mds/src/main/java/org/motechproject/mds/ex/object/DuplicateEntryException.java

```diff
@@ -0,0 +1,31 @@
+package org.motechproject.mds.ex.object;
+
+import org.motechproject.mds.ex.MdsException;
+
+import javax.jdo.JDODataStoreException;
+import java.sql.SQLIntegrityConstraintViolationException;
+
+/**
+ * Signals that it was not possible to update object instance from the provided data.
+ */
+public class DuplicateEntryException extends MdsException {
+
+    private static final long serialVersionUID = -6214111291407582493L;
+
+    public DuplicateEntryException(String entityName, Throwable cause) {
+        super("Can't insert entry: " + getCauseMessage(cause) + " in entity " + entityName + ". Thrown by: ", cause, "mds.error.objectUpdateError");
```

Comment:
Won't only the generic mds.error.objectUpdateError be displayed on the UI in case of this exception, from my understanding this ticket is about the UI message primarily.